client/src/components/APIDataTypes.ts

```diff
@@ -120,25 +120,16 @@ export type BuildingInfoRecord = {
   streetname: string;
   bldgclass: string;
   formatted_address: string;
-};
-
-export type BuildingInfoResults = {
-  result: BuildingInfoRecord[];
-};
-
-// TYPES ASSOCIATED WITH NYCHA STATS QUERY:
-
-export type NychaStatsRecord = {
-  /** The name of the NYCHA development */
+  /** The name of the NYCHA development (e.g. "SOTOMAYOR HOUSES"). NULL if building is not part of NYCHA. */
   development: string | null;
```

Comment:
Oof, since this is now part of general building info, do you think these NYCHA-specific fields should have slightly different names?  For instance, what if they were called `nycha_development`, `nycha_dev_evictions`, and `nycha_dev_unitsres`, just so clients (including e.g. Jade, who might be using this endpoint) aren't confused?

I guess that begs the question of why we don't put this information under a `nycha` object, which I would be an additional hassle for us right now, but might make things easier down the road... for example, we'd just have to test if `nycha` is `null` rather than having to assert that all three NYCHA-specific fields are non-null.  I'll leave those decisions up to you though.

Reply:
Good point about appending `nycha_` to each nycha-specific field. For clarification about your second point, we actually don't check if _all three_ NYCHA-specific fields are non-null, we only check `developments` as the other two can conceivably be NULL simply if there are no evictions or no res. units listed on PLUTO. So, I'm gonna file this as an issue now but since we only check to see if `developments` is populated it doesn't seem to add much complexity from my end IMO. Let me know if I'm missiong something